app/Models/EntryDAOPGSQL.php

```diff
@@ -29,6 +29,7 @@ public function commitNewEntries() {
 maxrank bigint := (SELECT MAX(id) FROM `' . $this->prefix . 'entrytmp`);
 rank bigint := (SELECT maxrank - COUNT(*) FROM `' . $this->prefix . 'entrytmp`);
 BEGIN
+	DELETE FROM `' . $this->prefix . 'entrytmp` AS entrytmp WHERE EXISTS (SELECT 1 FROM `' . $this->prefix . 'entry` AS entry WHERE entrytmp.id_feed = entry.id_feed AND entrytmp.guid = entry.guid);
```

Comment:
I am not sure of what happens if the prefix is empty and `entrytmp` is also used for the table name.
Just to avoid any risk, could you please use another name? (e.g. `e1` and `e2` or something you like).